website/docs/configure_flag/rollout/progressive.mdx

```diff
@@ -71,27 +68,30 @@ variationA = "A"
-| **`releaseRamp`** | It contains the time slot where we will progressively increase the percentage of the flag.<ul><li>**Before** the `start` date we will serve the `percentage.initial` percentage of the flag.</li><li>**Between** `start` and `end` we will serve a percentage of the flag corresponding to the actual time.</li><li>**After** the `end` date we will serve the `percentage.end` percentage of the flag.</li></ul><p>If you have no date in your `releaseRamp` we will not do any progressive rollout and use the top level percentage you have configured *(0% in our example)*.</p> |
-| **`percentage`**  | *(optional)*<br/>It represents the ramp of progress, at which level the flag starts (`initial`) and ends (`end`).<br/>**Default: `initial` = `0` and `end` = `100`**                                                                                                                                                                                                                                                                                                                                                                                               |
+| Field                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`percentage`**<br/>*(optional)*  | It represents the ramp of progress, at which level the flag starts (`initial`) and ends (`end`).<br/>**Default: `initial` = `0` and `end` = `100`**.                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
We probably need to change this table with explaining what should be in initial and end.